packager/packager.js

```diff
@@ -35,6 +35,14 @@ var parseCommandLine = require('./parseCommandLine.js');
 var webSocketProxy = require('./webSocketProxy.js');
 
 var options = parseCommandLine([{
+  command: 'protocol',
```

Comment:
what does it actually give us? if we wanted to use `https` we would have to set up certificates. Are there any other cases for this value to not be `http:`?